sbat.c

```diff
@@ -92,7 +74,7 @@ parse_sbat_section(char *section_base, size_t section_size,
 	list_for_each(pos, &csv) {
 		struct csv_row * row;
 		size_t i;
-		const char **ptrs[6] = {
+		const char **ptrs[] = {
```

Comment:
Right, you are removing the array size since is not really needed. Probably should be done in the previous patch.